Fix ability to build project

A recent [commit](https://github.com/greymatter-io/nifi-sdk/commit/47218c602775938e1f9f36fdd3192b99c3a4cfb8#diff-272d821466c9bb31632c6b078dddfdbaR68-R89) added a plugin defined as follows

```
			<plugin>
				<groupId>com.deciphernow</groupId>
				<artifactId>bouncycastle-maven-plugin</artifactId>
				<version>1.1.1</version>
				<configuration>
					<!--suppress UnresolvedMavenProperty -->
					<passphrase>${env.PGP_PASSPHRASE}</passphrase>
					<!--suppress UnresolvedMavenProperty -->
					<rings>${env.PGP_RINGS_ENCODED}</rings>
					<!--suppress UnresolvedMavenProperty -->
					<userId>${env.PGP_USER_ID}</userId>
				</configuration>
				<executions>
					<execution>
						<id>sign-artifacts</id>
						<phase>verify</phase>
						<goals>
							<goal>sign</goal>
						</goals>
					</execution>
				</executions>
			</plugin>
```

This is intended for some continuous integration with CircleCI for signing and pushing but is incomplete.
It is also blocking the project from being built locally.

In the meantime, since this plugin runs in the verify phase, which is included by the install phase, in theory should be able to skip it altogether with `mvn clean package`.  The goal being to build the nar, though it wont end up in the ~/.m2 local repo

Unfortunately, when this was attempted, after tests were run, a different error occurred
```
[INFO] Reactor Summary for nifi-data 1.0.1-SNAPSHOT:
[INFO] 
[INFO] nifi-data .......................................... SUCCESS [  2.346 s]
[INFO] nifi-data-processors ............................... SUCCESS [04:17 min]
[INFO] nifi-data-nar ...................................... FAILURE [  1.067 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  04:22 min
[INFO] Finished at: 2020-09-17T21:11:46Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-clean-plugin:3.1.0:clean (default-clean) on project nifi-data-nar: Failed to clean project: Failed to delete /home/lucasmoten/workspace/gocode/src/github.com/greymatter-io/nifi-sdk/gmd-sdk/nifi-data-nar/target/nifi-greymatter-data-nar-1.0.0.nar -> [Help 1]
```

--- a/gmd-sdk/pom.xml
+++ b/gmd-sdk/pom.xml
@@ -80,7 +80,7 @@
 				<executions>
 					<execution>
 						<id>sign-artifacts</id>
-						<phase>verify</phase>
+						<phase>deploy</phase>
 						<goals>
 							<goal>sign</goal>
 						</goals>

